### DISCUSSION TITLE
Weird elasticsearch issue

### DISCUSSION THREAD

**Discussion by adrianomic** [mastodon/mastodon#17563] [General]
```
NoMethodError (undefined method `fetch' for 0:Integer)
app/services/account_search_service.rb:92:in `from_elasticsearch'
app/services/account_search_service.rb:50:in `search_results'
...
```
My chewy version is 7.2
And elasticsearch version is 6.8.20
The issue comes from elasticsearch version or something else?

Look forward to your reply

**Comment by Gargron** [comment]
Probably from ElasticSearch version, yes. I think it needs to be 7 instead of 6.8

**Comment by adrianomic** [comment]
Thanks @Gargron 
It's resolved
